_plugins/generators/transcription_generator.rb

```diff
@@ -24,8 +24,10 @@ def generate(site)
           rescue
             message.data['transcription'] = nil
           ensure
-            local_copy_of_remote_file.close
-            local_copy_of_remote_file.unlink
+            if local_copy_of_remote_file
```

Comment:
Met with Taylor about this. We agreed that since this code is ran at build time, any risk for memory leakage is acceptable to take.